src/inc/internal/AppxManifestObject.hpp

```diff
@@ -317,10 +356,8 @@ namespace MSIX {
         HRESULT STDMETHODCALLTYPE GetApplications(IAppxManifestApplicationsEnumerator **applications) noexcept override;
         HRESULT STDMETHODCALLTYPE GetStream(IStream **manifestStream) noexcept override;
 
-        // IAppxManifestReader2
-        HRESULT STDMETHODCALLTYPE GetQualifiedResources(IAppxManifestQualifiedResourcesEnumerator **resources) noexcept override;

```

Comment:
I think you should keep it like this to be clear that this method comes from this interface, even though you can call this method via IAppxManifestReader3 or IAppxManifestReader4 because of inheritance

Reply:
Updated to reflect this.